README.md

```diff
@@ -376,6 +377,12 @@ Z = sqrt(tag<1>(X) * tag<1>(X) + tag<2>(Y) * tag<2>(Y));
 Here, the generated kernel will have one parameter for each of the vectors `X`
 and `Y`.
 
+### <a name="optimisation-keyword"></a>Optimisation Keywords
```

Comment:
'Optimization keywords' does not sound right. I think that 'pointer qualifiers' is a better name, but I am not about that either.